src/Concordium/Client/Types/Contract/Schema.hs

```diff
@@ -94,8 +94,32 @@ data ModuleSchema
   deriving (Eq, Show, Generic)
 
 -- |Create a getter based on the wasm version.
+-- Will first attempt to parse module schema as versioned and if it fails,
+-- it will attempt parsing based on the wasmVersion.
```

Comment:
```suggestion
-- Will first attempt to parse the module schema as versioned and if it fails,
-- it will attempt parsing based on the wasmVersion.
```